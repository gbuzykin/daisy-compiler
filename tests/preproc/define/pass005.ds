#define MACRO 1
#undef MACRO
#define MACRO 0

const a = MACRO;