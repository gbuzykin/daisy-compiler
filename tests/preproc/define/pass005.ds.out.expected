./preproc/define/pass005.ds:5:1: debug: token
 5 | const a = MACRO;
   | ^~~~~
./preproc/define/pass005.ds:5:7: debug: id: a
 5 | const a = MACRO;
   |       ^
./preproc/define/pass005.ds:5:9: debug: token
 5 | const a = MACRO;
   |         ^
./preproc/define/pass005.ds:5:11: debug: integer number: 0
 5 | const a = MACRO;
   |           ^~~~~
./preproc/define/pass005.ds:3:15: note: expanded from macro `MACRO`
 3 | #define MACRO 0
   |               ^
./preproc/define/pass005.ds:5:16: debug: token
 5 | const a = MACRO;
   |                ^
./preproc/define/pass005.ds: info: warnings 0, errors 0
